static/src/javascripts/projects/common/modules/commercial/permutive.js

```diff
@@ -2,14 +2,26 @@
 import { local } from 'lib/storage';
 
 const PERMUTIVE_KEY = `_papns`;
+const PFP_KEY = `_pdfps`;
```

Comment:
I think it'd be nice if `PFP_KEY` also included the word `PERMUTIVE`, to show that this is the permutive segments for PFP. Otherwise is makes it look like this file is doling things to PFP? This is a minor point!